plugins/SEO-Tool/js/main.js

```diff
@@ -0,0 +1,191 @@
+'use strict';
+
+document.addEventListener('DOMContentLoaded', async function() {
+
+	function clear() {
+		document.getElementById('mobile').innerHTML = '';
+		document.getElementById('desktop').innerHTML = '';
+		document.getElementById('gscStatus').innerHTML = '';
+		document.getElementById('gscIssues').innerHTML = '';
+	};
+
+	function colorClear(elemIds) {
+		elemIds.forEach((elemId, ind) => {
+			const element = document.querySelector(`[data-url='${elemId}']`);
+			element.children[0].children[0].classList.remove(...colors);
+		});
+	};
+
+	function newReport() {
+		clear();
+		reports = {};
+		luColors = {};
+		colorClear(pages);
+		colorClear(LUs);
+		storage.clear();
+		document.getElementById('loader').style.display = 'block';
+		reportGen();
+	};
+
+	function getDate(ts)
+	{
+		const date = new Date(ts), days = ["Sun", "Mon", "Tue", "Wed", "Thurs", "Fri", "Sat"];
+		let dateStrg = days[date.getDay()] + ", " + date.getDate() + "/" + ('0' + String(date.getMonth())).slice(-2) + "/" + date.getFullYear() + " " + ('0' + String(date.getHours())).slice(-2) + ":" + ('0' + String(date.getMinutes())).slice(-2) + ":" + ('0' + String(date.getSeconds())).slice(-2);
```

Comment:
Please use template literals for formatting the string. Also dateStrg can be a const